mwdb/web/src/components/Navigation.js

```diff
@@ -324,9 +320,13 @@ export default function Navigation() {
                                                 <React.Fragment>
                                                     <Link
                                                         className="btn btn-outline-success"
-                                                        to={`/profile/${auth.user.login}`}
+                                                        to="/settings"
                                                     >
-                                                        Profile
+                                                        <FontAwesomeIcon
+                                                            className="navbar-icon"
+                                                            icon={faCog}
+                                                        />
+                                                        Settings
```

Comment:
What about combining it with a logged user section and just display the user's login in badge instead of settings/profile.

Reply:
Implemented :+1: